doc/api/fs.md

```diff
@@ -2625,11 +2625,18 @@ Functions based on `fs.open()` exhibit this behavior as well:
 ## fs.opendir(path\[, options\], callback)
 <!-- YAML
 added: v12.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/30114
+    description: The `bufferSize` option was introduced.
 -->
 
 * `path` {string|Buffer|URL}
 * `options` {Object}
   * `encoding` {string|null} **Default:** `'utf8'`
+  * `bufferSize` {number} Size of the internal buffer when reading
```

Comment:
In the few descriptions of `bufferSize`, can you include the units. For example, number of directory entries vs. buffer size in bytes.

Reply:
Yup, rephrased to make it clear that this gives the # of directory entries. Thanks for pointing that out!